ui/plugins/eu.esdihumboldt.hale.ui/src/eu/esdihumboldt/hale/ui/io/schema/SourceSchemaActionUIAdvisor.java

```diff
@@ -0,0 +1,72 @@
+
+package eu.esdihumboldt.hale.ui.io.schema;
+
+import org.eclipse.ui.PlatformUI;
+
+import eu.esdihumboldt.hale.common.instance.model.DataSet;
+import eu.esdihumboldt.hale.common.instance.model.Instance;
+import eu.esdihumboldt.hale.common.instance.model.InstanceCollection;
+import eu.esdihumboldt.hale.common.instance.model.ResourceIterator;
+import eu.esdihumboldt.hale.common.schema.SchemaSpaceID;
+import eu.esdihumboldt.hale.common.schema.model.Schema;
+import eu.esdihumboldt.hale.ui.io.action.AbstractActionUIAdvisor;
+import eu.esdihumboldt.hale.ui.service.instance.InstanceService;
+import eu.esdihumboldt.hale.ui.service.schema.SchemaService;
+
+/**
+ * Source Schema action UI advisor.
+ * 
+ * @author Kapil Agnihotri.
+ */
+public class SourceSchemaActionUIAdvisor extends AbstractActionUIAdvisor<Schema> {
+
+	/**
+	 * @see eu.esdihumboldt.hale.ui.io.action.ActionUIAdvisor#supportsRemoval(java.lang.String)
+	 */
+	@Override
+	public boolean supportsRemoval(String resourceId) {
```

Comment:
For the source schema the instances are checked, but not the alignment.

Reply:
Is it possible to map source schema to the target schema even when the source data is not loaded?
I had an understanding that it is not and it was the reason the alignment was not checked.

If it is possible to map the source schema to the target schema when it has no source data available then is it a valid use case?

Reply:
> Is it possible to map source schema to the target schema even when the source data is not loaded?

Yes, you can certainly create mappings without data.

Reply:
Check for the alignment added for the source schema too.